xla/service/gpu/ir_emitter_unnested.h

```diff
@@ -377,6 +380,9 @@ class IrEmitterUnnested : public IrEmitter {
   // Container for async send/recv events shared by send/recv thunks.
   std::shared_ptr<SendRecvAsyncEvents> send_recv_events_;
```

Comment:
FWIW: I think it's a bad name for AsyncEvents and they should also be move under some thunk, the good example is `NcclCollectiveThunk::AsyncEvents` (this probaby should be `NcclP2pThunk::AsyncEvents` or something similar)

Reply:
Used CopyThunk::AsyncEvents as suggested.